Wdvh/standardcard/standardcard.py

```diff
@@ -0,0 +1,456 @@
+from pathlib import Path
+from re import findall
+
+from modules.CardType import CardType
+from modules.Debug import log
+
+class StandardTitleCard(CardType):
+    """
+    This class describes a type of CardType that produces the 'generic' title
+    cards based on Reddit user /u/UniversalPolymath. This card supports 
+    customization of every aspect of the card, but does not use any arbitrary
+    data.
+    """
+
+    """Directory where all reference files used by this card are stored"""
+    REF_DIRECTORY = Path(__file__).parent / 'ref'
+
+    """Characteristics for title splitting by this class"""
+    TITLE_CHARACTERISTICS = {
+        'max_line_width': 32,   # Character count to begin splitting titles
```

Comment:
```suggestion
        'max_line_width': 36,   # Character count to begin splitting titles
```
Can probably bump up the line width since you're using a smaller font